adage/analyze/api.py

```diff
@@ -423,3 +423,21 @@ class Meta:
             'node': ('exact', 'in', ),
             'gene': ('exact', 'in', ),
         }
+
+
+class ExpressionValueResource(ModelResource):
+    gene = fields.IntegerField(attribute='gene_id', null=False)
+    sample = fields.IntegerField(attribute='sample_id', null=False)
+
```

Comment:
So I guess we don't need to add the ``value`` field her, right? Tastypie will just add it automatically and we don't need to worry about converting from ForeignKey object to IntegerField like in the other fields, right?